docs/architecture/readMe.md

```diff
@@ -0,0 +1,16 @@
+# Architecture Decision Records
+
+This folder documents some of the architecturally significant decisions we've made during the Azure Service Operator project.
+
+Files are listed chronologically by month, and should contain the following sections:
+
+* Context
+* Decision
+* Status
+* Consequences
+* Experience Report
+* References
+
+For background information, check out [this Cognitect blog entry](https://www.cognitect.com/blog/2011/11/15/documenting-architecture-decisions?utm_source=pocket_mylist).
```

Comment:
minor: Not sure you want the `pocket_mylist` suffix here?

Reply:
Good catch. Updated.